grr/server/grr_response_server/gui/ui/lib/api/api_interfaces.ts

```diff
@@ -10,22 +10,115 @@
 
 export declare interface AnyObject {
   '@type'?: string;
-  [key: string]: undefined|null|string|number|boolean|object;
+  [key: string]: undefined | null | string | number | boolean | object;
+}
+
+/** Api RdfValueFieldDescriptor proto mapping. */
+export declare interface RdfValueFieldDescriptor {
+  readonly name?: string;
+  readonly doc?: string;
+  readonly friendlyName?: string;
+}
+
+/** Api RdfValueDescriptor proto mapping. */
+export declare interface RdfValueDescriptor {
+  readonly name?: string;
+  readonly doc?: string;
+  readonly fields?: ReadonlyArray<RdfValueFieldDescriptor>;
+}
+
+/** ApiAllRdfValues proto mapping. */
+export declare interface ApiAllRdfValues {
```

Comment:
The reflection stuff (ApiAllRdfValues, RdfValueDescriptor, RdfValueFieldDescriptor) is no longer used by the client details page right? If so, please remove it.